### PR TITLE
(SIMP-3034) Use polkit module

### PR DESCRIPTION
**Pull request by jeefberkey**
- Updated polkit setting to use the new defined type from the polkit module
- Added polkit and simplib dependency
- Package `ensure` is now set to the simp catalyst

SIMP-3034 #close

### CODE REVIEW

**Comment by jeefberkey** [comment]
Travis and spec tests will fail until simp/pupmod-simp-polkit#15 is merged.

File: manifests/dconf.pp [hunk 1]
```diff
@@ -139,15 +161,25 @@
...
+      ensure   => present,
+      priority => 10,
+      result   => 'yes'
+    }
+
+    polkit::authorization::basic_policy {
+      default:
+        * => $_policy_defaults;
+
+      'Allow anyone to shutdown system':
```
Comment (jeannegreulich) [inline manifests/dconf.pp]: It uses these names to create files.  Do you really want it have names with spaces?

**Comment by op-ct** [comment]
After simp/pupmod-simp-polkit#15 merge, tests fail in new and exciting ways  https://travis-ci.org/simp/pupmod-simp-gnome/jobs/222922961#L570:  :man_facepalming: 

```
 1) gnome::dconf on centos-7-x86_64 with default parameters should compile into a catalogue without dependency cycles
     Failure/Error: it { is_expected.to compile.with_all_deps }
       error during compilation: Evaluation Error: Error while evaluating a Resource Statement, Evaluation Error: Unknown function: 'inspect'. at /home/travis/build/simp/pupmod-simp-gnome/spec/fixtures/modules/polkit/manifests/authorization/rule.pp:20:3 at /home/travis/build/simp/pupmod-simp-gnome/spec/fixtures/modules/polkit/manifests/authorization/basic_policy.pp:109 
```

**Comment by jeefberkey** [comment]
Tests pass after a change in the polkit module: https://github.com/simp/pupmod-simp-polkit/pull/16

File: manifests/dconf.pp [hunk 2]
```diff
@@ -13,22 +13,44 @@
...
+      --------------------------------- ATTENTION ----------------------------------
+      |EOF
   }
-  file { $locks_dir :
+
+  # Create an array of directories based on profile names
+  $_dir       = $profile_list.map |$profile| { "/etc/dconf/db/${profile}.d" }
+  $_locks_dir = $profile_list.map |$profile| { "/etc/dconf/db/${profile}.d/locks" }
+
+  file { $_dir + $_locks_dir :
```
Comment (nick-markowski) [inline manifests/dconf.pp]: /etc/dconf/db is provided by the dconf rpm. We need to ensure it is installed before ensuring this file (and the one below it)

File: manifests/dconf.pp (hunk 1 above)
Comment (nick-markowski) [inline manifests/dconf.pp]: These are joined with underscores, which is acceptable

Comment (trevor-vaughan) [inline manifests/dconf.pp]: @jeannegreulich The underlying `polkit::authorization::rule` defined type makes everything path-safe (hopefully)

**Pull request merged**